### PR TITLE
Adds four Chainstack-sourced articles

### PR DESCRIPTION
**Pull request by TABASCOatw**
This PR adds four articles sources from Chainstacks developer tutorial docs to the Ethereum Development Tutorials section of ethereum.org.

This adds the following articles:
- Academic certificates with Truffle
- Indexing ERC20 token balances with Subgraphs
- Trust fund account with Remix
- Uncovering the power of eth_getBlockReceipts

Each article was hand-picked from Chainstack's "Web3 Decoded" developer tutorial series; all four articles cover largely different topics and should provide significant value to developers.

### CODE REVIEW

**Comment by TABASCOatw** [comment]
GM, can this be reviewed?

**Comment by TABASCOatw** [comment]
bump

**Comment by corwintines** [comment]
@TABASCOatw are you open to these being external tutorials?

File: src/content/developers/tutorials/academic-certificates-with-truffle/index.md [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+
+## Step-by-step
+
+### Create a public chain project
+
+See [Create a project](doc:manage-your-project#create-a-project).
+
+### Join the Ethereum Goerli testnet
+
+See [Join a public network](doc:manage-your-networks#join-a-public-network).
```
Comment (corwintines) [inline src/content/developers/tutorials/academic-certificates-with-truffle/index.md:46]: Part of the reason for this is there are a lot of broken links that you will need to go through and fix. For example here. 

Right now these tutorials are broken if we keep them internal.

I would prefer this be an external one as well given its tightly coupled to a project.

**Comment by TABASCOatw** [comment]
Good point, @corwintines- I've just changed all four to be external.

**Comment by TABASCOatw** [comment]
GM @corwintines, quick bump here. Made the requested changes and added an extra resource.

**Comment by TABASCOatw** [comment]
quick bump @corwintines

**Pull request closed without merging**